pkg/virt-operator/kubevirt.go

```diff
@@ -595,23 +595,15 @@ func (c *KubeVirtController) deleteAllInstallStrategy() error {
 
 func (c *KubeVirtController) getImageTag(kv *v1.KubeVirt) string {
 	if kv.Spec.ImageTag == "" {
-		if kv.Status.TargetKubeVirtVersion != "" {
-			return kv.Status.TargetKubeVirtVersion
-		} else {
-			return c.config.ImageTag
-		}
+		return c.config.ImageTag
```

Comment:
Functionally, this is really all that changed in this PR. The rest is all testing built around to verify the change.

When no imageTag (or imageRegistry) is set, we now follow the version of the operator as set by the OPERATOR_IMAGE environment variable on the virt-operator's deployment.

/cc @rthallisey @djzager - After this pr goes in, virt-operator will converge the KubeVirt infrastucture on whatever registry/imageTag is set on the OPERATOR_IMAGE env-var when no 'imageTag' is set on the CR

Reply:
And I didn't have the old version in mind, when assuming that it already works like we want it now.
What a nice little change :smile: